rats-devtools/src/python/rats/ci/_app.py

```diff
@@ -1,15 +1,9 @@
+import click
+
 from rats import apps, devtools  # type: ignore[reportAttributeAccessIssue]
```

Comment:
Both syntaxes work for `rats.apps` because it is defined in a different package.

Reply:
can you help me understand the behavior? i'd rather keep the common import format if we can fix the detection. is it just because devtools and apps are in different components?

Reply:
I'm not sure, but my guess is -
Probably more to do with them being under different folders than under different components, but I struggle to see a situation where you have two folders in the same package converge to the same module path, so in practice, yes - I think it's because devtools and apps are in different components that `from rats import apps` is allowed.

Reply:
ok let me see if i can find some misconfiguration to resolve this but otherwise i'll adopt your convention and maybe file some tickets with pyright. i assume maybe vscode is confused for the same reason pyright is

Reply:
couldn't get it to work but i asked for help here which seems like a relevant old ticket https://github.com/microsoft/pyright/issues/2882

Reply:
I tried restructuring this as the following while implementing the `CommandTree` structure we talked about last week, and didn't see any complaints from `ruff`:

```python
import rats.devtools as devtools
from rats import apps
```

What went sideways when you tried splitting these? (def agree about preferring to be able to reference them both the same way, regardless of if this works)

Reply:
it sounds like a bug of vscode/pyright that it does not allow these to be imported on one line